clio-client/src/test/scala/org/broadinstitute/clio/client/commands/ParsersSpec.scala

```diff
@@ -117,4 +117,79 @@ class ParsersSpec extends BaseClientSpec {
     }
     ignoreDeleted should be(true)
   }
+
+  it should "double quotes on the ends of String inputs should cause an error" in {
```

Comment:
not a big deal, but it just reads better if it were `it should "error out when there are double quotes on the ends of String inputs" in {`